controller/persistence/migration_v13.go

```diff
@@ -0,0 +1,170 @@
+package persistence
+
+import (
+	"github.com/openziti/foundation/storage/boltz"
+	log "github.com/sirupsen/logrus"
+	"math"
+)
+
+func (m *Migrations) createTunnelerClientV2dot0ConfigType(step *boltz.MigrationStep) {
+	configName := "ziti-tunneler-client.v2.0"
+	clientConfigV2dot0TypeId := "g7cIWbcGg"
+	clientConfigTypeV2dot0 := &ConfigType{
+		BaseExtEntity: boltz.BaseExtEntity{Id: clientConfigV2dot0TypeId},
+		Name:          configName,
+		Schema: map[string]interface{}{
```

Comment:
Might have been easier to write this in JSON and then parse it to a map[string]interface{}.

Reply:
This is what I did. The raw schema is in this PRs description. I was wondering if there was a good place to check in the raw schema file also?